content/shmem_iput.tex

```diff
@@ -69,9 +69,8 @@
     on specified remote \ac{PE}. Both strides, \VAR{dst} and \VAR{sst}, must be
     greater than or equal to \CONST{1}. The routines return when the data has
     been copied out of the \VAR{source} array on the local \ac{PE} but not
-    necessarily before the data has been delivered to the remote data object.
-    If the context handle \VAR{ctx} does not correspond to a valid context,
-    the behavior is undefined.
+    necessarily before the data has been delivered to the remote data object. Use
+    \FUNC{shmem\_quiet} to force completion of all \FUNC{iput} transfers.
```

Comment:
I don't see a statement like this for `put` operations.  I wonder if the change below to quiet is sufficient?  Also, looks like we lost a sentence as part of this change.

Reply:
@jdinan I was trying to keep consistent with shmem_p. Also, we have a similar sentence in nonblocking puts. I did not realize that we did not have a sentence like that in shmem_put. I added one sentence to shmem_put.